### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/zh/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#535] [Announcements]
# https://plugin-comment2.vuejs.press/zh/demo.html

为了让你更好的了解如何使用插件，我们使用 vuepress-plugin-comment2 和 @vuepress/theme-default 制作了一个案例。 源代码 (https://github.com/vuepress-theme-hope/vuepress-theme-hope/tree/main/demo/comment2/); 预览 (h...

https://plugin-comment2.vuejs.press/zh/demo.html

**Comment by OnePeiece** [comment]
你好啊

**Reply by OnePeiece** [reply]
我知道你是谁

**Reply by OnePeiece** [reply]
你这是在干什么

**Reply by OnePeiece** [reply]
好滴。亲，这个大大滴好用

**Comment by OnePeiece** [comment]
阿大

**Comment by lattechen925** [comment]
能写一个vuepress-theme-hope 主题 使用 vuepress-plugin-comment2 的案例吗？

**Reply by Mister-Hope** [reply]
内置的

**Reply by lattechen925** [reply]
是开启之后，每个页面底部都有评论组件吗？

**Reply by Mister-Hope** [reply]
主页和博客文章列表页没有

**Comment by Kolentoo** [comment]
这插件 vuepress2 打包的时候报错啊

**Reply by JasonSubMara** [reply]
我的也是，而且按照教程配置了之后，没有出现评论相关控件，您解决了么？

**Reply by Mile-Away** [reply]
是的，我也遇到了没有评论相关控件的问题，包括 readingtime 这个插件也没有出现相应的前端展示

**Reply by Mister-Hope** [reply]
![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/33315834/c0b96c35-0d61-4cab-9f2c-6e198a84b54c)

**Comment by yangkun19921001** [comment]
test

**Comment by zhanglinweigithub** [comment]
您好，请问一下，vuepress-theme-hope主题使用了Giscus评论功能，如何添加浏览量？

**Reply by zhanglinweigithub** [reply]
只有 Waline 才可以用浏览量吗？

**Reply by Mister-Hope** [reply]
Giscus 没有浏览量的功能

**Comment by zou88288** [comment]
高级

**Reply by zou88288** [reply]
你
